azure-pipelines.yml

```diff
@@ -1,39 +1,38 @@
-trigger: none
-
-# trigger:
-#   branches:
-#     include:
-#       - main
-#   paths:
-#     include:
-#       - public
-#       - packages
-
 pool:
   name: DevCentral-Skyline
 
+trigger:
+  branches:
+    include:
+      - ni/current
```

Comment:
Hmm, I’m wondering if we also need to trigger off release branches. I don’t think the policy doc mentioned a plan for release branching.

Reply:
Yeah it would be good to figure out a release branching strategy (if we even need one). We can always add that trigger later too.